### ISSUE TITLE
Receiving text does not work.

### ISSUE THREAD

**Issue by SnipeXandrej** [vicr123/QNearbyShare#1]
As of now, sending text from Android's clipboard to my computer does not work properly. It succeeds on my phone, but on my computer it is stuck at `Starting transfer...`

My guess is that sending text was simply not implemented, because QNearbyShare was mainly coded to be used for transferring files, right?
Besides that, this is a really cool project! :)

**Comment by vicr123** [comment]
Yeah, I noticed the text stuff and made a mental note on figuring out what to do with it later but I haven't really thought about that yet 😅

I could definitely copy text to the clipboard and open a URL in the browser but the thing that kind of makes it weird is that you need to be discoverable first.

Maybe I should just send the contents of the message over DBus and let the client app deal with it?

**Comment by SnipeXandrej** [comment]
I was thinking about just displaying the copied text/url in terminal and/or displaying it in terminal and copying it to the clipboard at the same time, at least for the time being, until something better comes to mind 😅.
